src/server/pfs/server/api_server.go

```diff
@@ -623,6 +623,14 @@ func (a *apiServer) GlobFileV2(req *pfs.GlobFileRequest, server pfs.API_GlobFile
 	return errV2NotImplemented
 }
 
+func (a *apiServer) InspectFileV2(ctx context.Context, req *pfs.InspectFileRequest) (*pfs.FileInfoV2, error) {
+	return nil, errors.Errorf("v2 method not implemented")
```

Comment:
`InspectFileV2` and `WalkFileV2` should be using the error variable you setup (`errV2NotImplemented`).